ui/package.json

```diff
@@ -18,6 +19,7 @@
     "react-dom": "18.2.0",
     "react-query-subscription": "2.0.0-beta.1",
     "react-router-dom": "^6.10.0",
+    "react-tooltip": "^5.11.1",
```

Comment:
you do not need to add either matrial-iu/core, nor react-tooltip.
Please remove them